src/wallet/signer.rs

```diff
@@ -667,6 +667,15 @@ pub struct SignOptions {
     ///
     /// Defaults to `false` which will only allow signing using `SIGHASH_ALL`.
     pub allow_all_sighashes: bool,
+    /// Whether partial_sigs in input should be removed when the psbt inputs are signed in finalizing psbt.
+    ///
+    /// Defaults to `true` which will remove partial_sigs in inputs after signing.
```

Comment:
Small doc nit. This sentences can be simplified a little. 
```suggestion
    /// Whether to remove partial_sigs from psbt inputs while finalizing psbt.
    ///
    /// Defaults to `true` which will remove partial_sigs after finalizing.
```